.github/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+Fixes #XXXX.
+
+(Replace XXXX with the id of the issue fixed in this PR. Remove this line if there is no corresponding
+issue. Don't reference the issue in the title of this pull-request.)
+
+Add tags to your PR if you are a committer (only committers have the right to add tags). Add [Design Review] tag
```

Comment:
Minor nit: I think the "only committers have the right to add tags" is implied by the "if you are a committer" part and can be left out.

Maybe also we should add a blurb to otherwise expect a committer to likely/eventually tag your PR for you as we work our way through to review things?

Reply:
A blurb for the PR author (who is not a committer, in this case), or for the reviewing committers?

There is a "PR Management" section in the [checklist draft](https://docs.google.com/document/d/17EEKT6fih9Dd5NfXjBoECcKbVp1eOB2vb3jKqTF9pPc/edit). Perhaps we should extract it as a separate document "PR action item checklist for a committer" and publish to the wiki or the repository more promptly.

Reply:
I think it's a definitely good idea to have a checklist, but I'm not sure if anyone tries to read them all if the checklist is 10+ pages long. Maybe the current checklist draft is too detailed. It would be better to be compact as much as possible.